### PR TITLE
add conv bn fuse pass

### PR DESCRIPTION
**Pull request by zzpmiracle**

### CODE REVIEW

File: pytorch_blade/torch_blade/pass_manager.py [hunk 1]
```diff
@@ -319,6 +319,8 @@ def _optimize_common(c_module, static_shape=False):
         graph = c_module.forward.graph
         _jit_pass_remove_nograd(graph)
         _jit_pass_freeze_requires_grad(graph)
+        if hasattr(torch._C, "_jit_pass_fold_frozen_conv_bn"):
+            torch._C._jit_pass_fold_frozen_conv_bn(graph)
```
Comment (fortianyou) [inline pytorch_blade/torch_blade/pass_manager.py:323]: How about adding an unittest?

**Review by fortianyou — APPROVED** [review]
LGTM

File: pytorch_blade/tests/tensorrt/test_onnx.py [hunk 2]
```diff
@@ -67,5 +68,38 @@ def forward(self, x):
...
+            def forward(self, x):
+                x = F.relu(self.bn(self.conv(x)))
+                return x
+
+        script_module = torch.jit.script(Net()).eval()
+        inputs = torch.ones([1, 3, 20, 20])
+        script_module = exporter.export(script_module, model_inputs=(inputs,))
+        script_module = pass_manager._optimize_common(script_module._c)
+        onnx_graph, _ = pass_manager._jit_pass_lower_to_onnx(script_module.forward.graph)
+        print(onnx_graph)
```
Comment (fortianyou) [inline pytorch_blade/tests/tensorrt/test_onnx.py:88]: Nit: should not print directly in the ut

Comment (zzpmiracle) [reply pytorch_blade/tests/tensorrt/test_onnx.py]: Done.

**Review by fortianyou — APPROVED** [review]

**Pull request merged**